extensions/resource-deployment/src/ui/notebookWizard/notebookWizard.ts

```diff
@@ -58,7 +59,9 @@ export class NotebookWizard extends WizardBase<NotebookWizard, NotebookWizardPag
 			const notebook = await this.prepareNotebookAndEnvironment();
 			await this.openNotebook(notebook);
 		} catch (error) {
-			vscode.window.showErrorMessage(error);
+			console.log(error);
+			vscode.window.showErrorMessage(getErrorMessage(error));
+			throw error;
```

Comment:
What's catching this error?

Reply:
It gets caught by the onOk, onGenerateScript handlers and rethrown from there for vscode wizard code to abort the closing of the wizard.

Reply:
What I said above about vscode aborting the closing of the wizard is a conjecture from my observation that if I throw that stops the wizard from closing and if I don't that the wizard is closed. I have not actually looked into the vscode codebase for it. I will add a comment there to explain why we are re-throwing there.

Reply:
I guess that's ok - but just FYI the wizard code is all ours. Not VS Code. So it'd be good to understand what's happening here if you don't have that now. The behavior matches what I'd expect in general but know exactly what's going on is usually a good idea when you're throwing errors.

Reply:
You are right, found the code, here it is:
https://github.com/microsoft/azuredatastudio/blob/649f18b6f39403f6744fc7207569c2486a781fbf/src/sql/workbench/services/dialog/browser/wizardModal.ts#L85-L85
https://github.com/microsoft/azuredatastudio/blob/649f18b6f39403f6744fc7207569c2486a781fbf/src/sql/workbench/services/dialog/browser/wizardModal.ts#L259-L268
https://github.com/microsoft/azuredatastudio/blob/649f18b6f39403f6744fc7207569c2486a781fbf/src/sql/workbench/services/dialog/browser/wizardModal.ts#L179-L180



So, my conjecture was correct.